templates/templates/navigation-enterprise-h.html

```diff
@@ -130,9 +130,13 @@ <h4 class="u-show--small p-muted-heading u-hide--medium u-hide--large">
       <!-- Public cloud -->
       <div class="col-2">
         <h4 class="p-muted-heading">
-          <a href="/cloud/public-cloud">Public cloud&nbsp;&rsaquo;</a>
+          <a href="/cloud/public-cloud">CLOUD&nbsp;&rsaquo;</a>
```

Comment:
CSS will handle transforming the text, so no need to capitalise - LGTM otherwise!
```suggestion
          <a href="/cloud/public-cloud">Cloud&nbsp;&rsaquo;</a>
```